src/Init/Meta.lean

```diff
@@ -115,7 +115,7 @@ def getRoot : Name → Name
 
 @[export lean_is_inaccessible_user_name]
 def isInaccessibleUserName : Name → Bool
-  | Name.str _ s   => s.contains '✝︎' || s == "_inaccessible"
+  | Name.str _ s   => s.contains '✝' || s == "_inaccessible"
```

Comment:
TODO: here it crashed when changing it.

Reply:
Maybe it's not a single character to lean's lexer?

Reply:
Indeed, it says "missing end of character literal". This makes me think that this change to `✝︎` might be a bad idea even though it probably works the way the PR is rn...

Reply:
That’s the same error you get with
```lean
#check 'ab'
```

So I’d say

* That error message should be improved (Maybe parse `'ab'` like `"ab"` and give a helpful error if it’s not a single unicode codepoint.
* For this PR, the `s.contains '✝︎'` has to become `s.hasSubString '✝'`, or (a bit hacky, leave comment) leave the code as it, since the old `✝` character appears in the new string `✝`, so this check will still work.